internal/monitors/telegraf/common/accumulator/accumulator.go

```diff
@@ -1,12 +1,21 @@
 package accumulator
 
 import (
+	"fmt"
 	"time"
 
-	"github.com/signalfx/golib/datapoint"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
 	"github.com/signalfx/signalfx-agent/internal/monitors/telegraf/common/emitter"
 )
 
+func getTime(times []time.Time) time.Time {
+	if len(times) > 0 {
+		return times[0]
```

Comment:
The first item is the latest time available?

Reply:
yeah this is what telegraf does too.  It addresses a design issue from their old metric model.